packages/frontend/src/App.tsx

```diff
@@ -33,6 +33,13 @@ const queryClient = new QueryClient({
     defaultOptions: {
         queries: {
             refetchOnWindowFocus: false,
+            onError: async (result) => {
+                // @ts-ignore
+                const { error: { statusCode } = {} } = result;
+                if (statusCode === 401) {
```

Comment:
could we avoid this ts-ignore if we did something like this `if(result.error?.statusCode === 401)` ?

Reply:
done